src/transforms/code-breakify-transform.js

```diff
@@ -17,8 +18,17 @@ function breakify(content) {
     }
   }
 
+  // Расстановка переносов по спецсимволам
   for (const symbol of symbols) {
-    content = content.replaceAll(symbol, (match) => `<wbr>${match}<wbr>`)
+    const startsWithSymbol = new RegExp(`^[\\${symbol}]`, '')
+    const firstSymbolWithinTags = new RegExp(`^(<wbr>)[\\${symbol}](<wbr>)`, '')
+
+    if (!startsWithSymbol.test(content)) {
+      content = content.replaceAll(symbol, `<wbr>${symbol}<wbr>`)
+    } else {
+      // Исключение для переносов первого спецсимвола в слове
+      content = content.replaceAll(symbol, `<wbr>${symbol}<wbr>`).replace(firstSymbolWithinTags, `${symbol}`)
```

Comment:
```suggestion
    const firstSymbolWithinTags = new RegExp(`^(<wbr>)[\\${symbol}](<wbr>)`, '')

    content = content.replaceAll(symbol, `<wbr>${symbol}<wbr>`)

    // Исключение для переносов первого спецсимвола в слове
    if (startsWithSymbol.test(content)) {
      content = content.replace(firstSymbolWithinTags, `${symbol}`)
```

Кажется, так лаконичнее.

Reply:
Да, выглядит лаконичнее! Но проблема в том, что если сперва мы размечаем `content`, а уже потом вводим исключение, то имена методов будут начинаться с конструкции `<wbr>`, а не со спецсимвола, поэтому `startsWithSymbol` будет всё время `false`, и исключение не сработает.

По этой причине я размечал `content` после проверки, чтобы этот регэксп работал.

Но можно и правда сильно сократить, не потеряв смысл. Щас подумал, а как на счёт просто махнуть `startsWithSymbol` на `firstSymbolWithinTags` в исключении? Тогда и лаконичнее станет, и всё заработает, и кода меньше! 😀

Reply:
Давайте пробовать уже! 🚀